data/estimate.py

```diff
@@ -0,0 +1,11 @@
+def mysql_estimate_row_count(model_cls, db):
+    """ Uses the information_schema to retrieve the row count for a table. """
+    query = "SELECT table_name, table_rows FROM information_schema.tables WHERE table_schema = DATABASE() AND table_name = %s"
+    cursor = db.execute_sql(query, (model_cls._meta.table_name,))
+    res = cursor.fetchone()
```

Comment:
will this raise an error if there isn't a result? are you ok with that?

Reply:
It will and I am, as it shouldn't happen in a working database